main/src/main/scala/sbt/Main.scala

```diff
@@ -898,6 +898,9 @@ object BuiltinCommands {
       .getOpt(Keys.minForcegcInterval)
       .getOrElse(GCUtil.defaultMinForcegcInterval)
     val exec: Exec = exchange.blockUntilNextExec(minGCInterval, s1.globalLogging.full)
+    if (exec.source.fold(true)(_.channelName != "console0")) {
+      s1.log.info(s"Running remote command: ${exec.commandLine}")
```

Comment:
```suggestion
      s1.log.info(s"received: ${exec.commandLine}")
```

Reply:
I don't mind changing Running to received, but I think it makes sense to emphasize that it was a remote command.

Reply:
Cool. https://github.com/sbt/sbt/pull/5319/commits/73235d12e0dc4cea61996c4ff7d1ac27f75dd487